docs/bouncing.html

```diff
@@ -0,0 +1,85 @@
+</style>
+
+<body class=has-nav>
+
+<nav id=nav>
+    <ul>
+        <h1 class=title><a href="https://github.com/livewires/you-win">you-win</a></h1>
+
+        <li class=page><a href="install">Install</a>
+        <li class=page><a href="1-numbers">#1: Numbers</a>
+        <li class=page><a href="2-animation">#2: Animation</a>
+        <li class=page><a href="3-arrays">#3: Arrays</a>
+        <li class=page><a href="4-functions">#4: Functions</a>
+        <li class=page><a href="jump">Game</a>
+        <li class=page><a href="reference">API Reference</a>
+
+        <a class=page-as-pdf href="pdfs/bouncing.pdf">Page as PDF ↓</a>
+        <a class=logo href="http://livewires.org.uk/"></a>
+    </ul>
+</nav>
+
+<div class=prose>
+<h1 id="bouncing">Bouncing</h1>
+<p>This sheet explains how to make a sprite move around, and bounce off different corners of the screen.</p>
+<h2 id="making-a-sprite">Making a sprite</h2>
+<blockquote>
+<p><strong>Skip this step if you already have a sprite!</strong></p>
+</blockquote>
+<pre><code class="language-js">var ball = new Sprite
+ball.costume = &#39;<img class="emojione" alt="🏀" title=":basketball:" src="https://cdn.jsdelivr.net/emojione/assets/3.1/png/32/1f3c0.png"/>&#39;</code></pre>
+<h2 id="moving-the-ball">Moving the ball</h2>
+<p>For our ball to bounce, we need to “remember” which edge of the screen we bounced off last. Otherwise the computer won’t know which way to move the ball.</p>
+<p>So we introduce two variables – one for the movement in the X direction (left or right), and one for the movement in the Y direction (up or down).</p>
+<pre><code class="language-js">var velX = -2
+var velY = 0</code></pre>
+<p>We’ll call them <code>velX</code> and <code>velY</code>, short for “velocity” (which means speed). We’ve chosen <code>-2</code> and <code>0</code> to start the ball moving off to the left.</p>
+<p>Now, let’s move the ball a little bit each frame.</p>
+<pre><code class="language-js">ball.forever(() =&gt; {
+    ball.posX += velX
```

Comment:
Do you explain what `+=` does?

Reply:
Yes! https://livewires.github.io/you-win/04-forever#spinning